src/garage/sampler/off_policy_vectorized_sampler.py

```diff
@@ -93,15 +95,19 @@ def obtain_samples(self, itr, batch_size=None, whole_paths=True):
 
         while n_samples < batch_size:
             policy.reset(completes)
-            input_obses = self.algo.env_spec.observation_space.flatten_n(obses)
-            obs_normalized = tensor_utils.normalize_pixel_batch(
-                self._env_spec, input_obses)
+            obs_space = self.algo.env_spec.observation_space
+
+            if not isinstance(obs_space, akro.Image):
+                input_obses = obs_space.flatten_n(obses)
+            else:
+                input_obses = obses
+
```

Comment:
@krzentner is this the expected behavior? Just want to make sure this change doesn't break anything.

Reply:
I think it's better to fix this in the policy. (i.e. assume that inputs into the policy *might* be flattened already).

Reply:
Fixed. The sampler now flattens all inputs, and CNN policies unflatten them.